cmd/oras/internal/display/status.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package display
+
+import (
+	"context"
+
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+// PreCopyStatus returns a tracking function for uploading status.
+func PreCopyStatus(skip func() bool) func(context.Context, ocispec.Descriptor) error {
```

Comment:
Just curious are there any scenarios for the `skip` func?

Reply:
Changed to verbose since it's a common use case